includes/multi-currency/class-frontend-currencies.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * WooCommerce Payments Multi Currency Frontend Currencies
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Multi_Currency;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class that formats Multi Currency currencies on the frontend.
+ */
+class Frontend_Currencies {
+	/**
+	 * Multi-Currency instance.
+	 *
+	 * @var Multi_Currency
+	 */
+	protected $multi_currency;
+
+	/**
+	 * Multi-Currency currency formatting map.
+	 *
+	 * @var array
+	 */
+	protected $currency_settings = [];
```

Comment:
This is probably overkill, but what do you think about creating a class to represent this rather than using an array?

Reply:
I also think that at this point it might be overkill. Using the array structure might make it easier if people need to use the `wcpay_multi_currency_<currency>_settings`. I'm happy either way though, so let me know if you'd rather implement a class now 🙂

Reply:
That makes sense, I'm happy to keep it as an array as you said I think it makes it simpler for people who may need to use the hooks. :)